cmd/queue/main.go

```diff
@@ -63,7 +63,7 @@ const (
 	// reportingPeriod is the interval of time between reporting stats by queue proxy.
 	reportingPeriod = 1 * time.Second
 
-	unixSocketPath = "queue.sock"
+	unixSocketPath = "/queue.sock"
```

Comment:
we should probably call this `unixSocketName` since this isn't the path any more

Reply:
Yes, and I changed it to `var unixSocketPath        = filepath.Join(os.TempDir(), "queue.sock")`.